src/index.ts

```diff
@@ -258,6 +259,21 @@ program
         }
     });
 
+const devicesGroup = program.command('devices').alias('d').description('Operations on devices');
+
+devicesGroup
+    .command('list')
+    .option('--json', 'Output in JSON format')
+    .description('Lists all registered devices that can access to your account')
+    .action(listAllDevices);
+devicesGroup
+    .command('remove')
+    .option('--all', 'remove all devices (dangerous)')
+    .option('--other', 'remove all other devices')
+    .argument('[device ids...]', 'ids of the devices to remove')
+    .description('Unregisters a list of devices. Unregistering the CLI will implies a reset')
```

Comment:
```suggestion
    .description('Unregisters a list of devices. Unregistering the CLI will implies doing a "dcli reset"')
```

it could be confusing to say reset, I don't want people to think they have to reset their account ahah

Reply:
True, do you want me to rename the reset command as well ?
Maybe "clean-local-data" ?

Reply:
that's something we can consider yes

wipe or clean?